### PR TITLE
Create Professional Services Link

### PR DESCRIPTION
**Pull request by beardofedu**
## Overview
**TL;DR**
This pull request creates a link at the boot of the `simple-class` layout to an **About Professional Services** page that links to our paid engagements. 

Example of link
image

Example of about page
image

### Questions
1. Are we happy with the wording?
1. Are we ok with inline styles or do we want to create a CSS class like adults?
1. Are we happy with the current placement of the link?

### Next Steps
Depending on the answers to the above questions:
1. Create a new style
1. Modify the wording of the About page
1. Modify the placement of the link

### Review
@hectorsector for 👀  as you opened an issue regarding this functionality
@github/services-training since this was discussed by the team

### CODE REVIEW

**Comment by brianamarie** [comment]
> Are we happy with the wording?

I am! 🙌 

> Are we ok with inline styles or do we want to create a CSS class like adults?

I mean...I think it's OK to have inline styles, especially when thinking this will all hopefully be redesigned in the not too distant future when moving to Guides+. 🌠 

> Are we happy with the current placement of the link?

I think the link is good. But...do you think we could put a "contact us" form in the "about us" page, like in the other services pages? @JasonEtco Do you know if this is possible?

**Comment by crichID** [comment]
@beardofedu @brianamarie I :heart: how this collaboration thing works :wink:.

> do you think we could put a "contact us" form in the "about us" page, like in the other services pages?

This reminds me of the footer of the [opensource guides](https://opensource.guide/) that I :heart: for a few reasons:

screen shot 2017-07-18 at 8 42 08 am

1. I love how they openly call for contributions in the footer of every page. If a visitor sees something that could be improved, they can immediately take action.
2. They have the little form to sign up for updates. 
3. They specifically call out that it was Coded by GitHub **and friends**. 

As for the full "contact us" form, could we handle that like they do the **fine print** on the opensource guides and have a link that says "Contact GitHub Training" going back to the contact page on services.github.com?

**Comment by hectorsector** [comment]
✨ @beardofedu

**Comment by beardofedu** [comment]
@crichID  

> As for the full "contact us" form, could we handle that like they do the fine print on the opensource guides and have a link that says "Contact GitHub Training" going back to the contact page on services.github.com?

So do you just want a link in the footer that says Contact GitHub Training? 

In my latest commit I changed the wording of the About page to send you to the Contact form as opposed to just listing our email.

**Comment by beardofedu** [comment]
> 
image

Is going to be tackled in another PR per @crichID so, this small ship is ready once @JasonEtco finishes messing with the spacing of the footer.

**Comment by JasonEtco** [comment]
@beardofedu sorry for taking so long, should be good to merge now 🚢

**Comment by crichID** [comment]
This one was sitting and waiting to be :ship:ed so I went ahead and pressed the green button :tada:
